integration_tests/e2e/induction/createShortQuestionSetInduction.cy.ts

```diff
@@ -0,0 +1,38 @@
+import OverviewPage from '../../pages/overview/OverviewPage'
+import Page from '../../pages/page'
+import HopingToGetWorkValue from '../../../server/enums/hopingToGetWorkValue'
+import HopingToWorkOnReleasePage from '../../pages/induction/HopingToWorkOnReleasePage'
+import ReasonsNotToGetWorkPage from '../../pages/induction/ReasonsNotToGetWorkPage'
+import ReasonNotToGetWorkValue from '../../../server/enums/reasonNotToGetWorkValue'
+
+context('Create a short question set Induction', () => {
+  beforeEach(() => {
+    cy.signInAsUserWithEditAuthorityToArriveOnPrisonerListPage()
+  })
+
+  it('should create a long question set Induction', () => {
```

Comment:
Minor copy and paste typo 😁 
Should be `should create a short question set Induction`